components/comment.js

```diff
@@ -197,7 +200,7 @@ export default function Comment ({
               />
               )
             : (
-              <div className={styles.text}>
+              <div className={styles.text} ref={contentContainerRef}>
```

Comment:
Mark the DOM tree root where the item's text can be found, so we can check to see if a selection is made in the item when doing a quote reply, not just any selection in the entire page.